tethys_apps/models.py

```diff
@@ -370,23 +500,153 @@ def get_value(self):
         if self.type == self.TYPE_UUID:
             return uuid.UUID(self.value)
 
+class CustomJSONSetting(CustomSetting):
```

Comment:
Rename this to JSONCustomSetting

Reply:
CustomJSONSetting has been renamed JSONCustomSetting